### DISCUSSION TITLE
Issues created from issue forms template should not render empty fields

### DISCUSSION THREAD

**Discussion by jpraet** [community/community#4491] [Projects and Issues]
When the user did not provide a value for an optional field in the issue form, this gets rendered in the issue as _No response_.

![afbeelding](https://user-images.githubusercontent.com/2212615/124380828-49cd7d00-dcbf-11eb-9e97-b0babbc4f39f.png)

In my opinion, this is not very useful and those fields should not be displayed.

What do you think?

**Comment by lukehefson** [comment]
Hey @jpraet! We know that _some_ maintainers find the `No response` input as useful because they can scan a submitted issue and quickly determine what sections a submitter left out.

However, I can see how that can also create extra noise for some maintainers. Perhaps it’s something we could add an option for in the future 🤔

**Reply by BoD** [reply]
An option would be great! 
In our repo we initially had an "Anything else?" field to incite submitters to provide more context and details, etc. This ended up generating noise since most submitters left it empty. I still think such field could sometimes be useful, but only if it can be hidden when not provided.

**Comment by Ari-Kishon** [comment]
Any new on this?

It seems like a pretty trivial feature that would improve readability immensely.

Ideally I should be able to prevent the block from rendering if it has no response
